### ISSUE TITLE
Live tv exists automatically

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/mythbox#59]
```
What steps will reproduce the problem?
1. select watch tv
2. select channel
3. channel starts then exits in about the time the buffer took to fill

What is the expected output? What do you see instead?

Expected to continue to watch the channel.

What version of the product are you using? On what operating system?

Beta svn 1260. Leopard.

Please provide any additional information below.
```

Original issue reported on code.google.com by `waterfue...@gmail.com` on 26 Sep 2009 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
Can you please drop me an email @ analogue@yahoo.com and I'll send you a newer
version to test. tnx!
```

Original comment by `semir.pa...@gmail.com` on 2 Oct 2009 at 9:28

**Comment by GoogleCodeExporter** [comment]
```
Fixed in svn 1510 for 0.21 backends
```

Original comment by `semir.pa...@gmail.com` on 21 Oct 2009 at 1:16
- Changed state: **Fixed**
